apptainer/README.md

```diff
@@ -0,0 +1,60 @@
+# Apptainer
+
+Apptainer (formerly Singularity) is an [open-source](https://github.com/apptainer/apptainer) container platform that allows users to build and run containers in a simple, portable and secure way. It is part of the [Linux Foundation](https://apptainer.org/news/community-announcement-20211130/).
+
+## Install Apptainer
+
+You will need a linux system to install and use apptainers. You can install Apptainer either from source or from pre-built packages. Refer to installation instructions provided [here](https://apptainer.org/docs/admin/main/installation.html). For example, on Ubuntu based system, follow these instructions.
+
+```bash
+sudo apt update
+sudo apt install -y software-properties-common
+sudo add-apt-repository -y ppa:apptainer/ppa
+sudo apt update
+sudo apt install -y apptainer
+apptainer version
+```
```

Comment:
rather than giving instructions, it might just be better to refer to the apptainer docs in case the BKM for installation changes.

Reply:
removed.